src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days >= 7) {
+    sum = days * 40 - 50;
+  } else if (days >= 3 && days < 7) {
```

Comment:
If you will return value from `if`'s instantly, you can remove unnecessary `else`'s

Reply:
Pay attention to all the comments, and remove unnecessary `else`'s

Reply:
And you don't need this additional check, cause you check the opposite of this condition in previous if clause
```suggestion
  } else if (days >= 3) {
```